apps/cpp_rpc/README.md

```diff
@@ -37,7 +37,15 @@ This folder contains a simple recipe to make RPC server in c++.
   # Path to the desired C++ cross compiler
   set(CMAKE_CXX_COMPILER /path/to/cross/compiler/executable)
 ```
-- If linking against a custom device OpenCL library is needed, in the config specify the path to the OpenCL SDK containing the include/CL headers and lib/ or lib64/libOpenCL.so:
+- If you need to build cpp_rpc with OpenCL support, specify variable `USE_OPENCL` in the config:
+  ```
+  set(USE_OPENCL ON)
+  ```
+  In this case [OpenCL-wrapper](../../src/runtime/opencl/opencl_wrapper) or OpenCL installed to your system will be used.
```

Comment:
> or OpenCL installed

did you mean OpenCL SDK?
Do we consider OpenCL SDK flow at all?

Reply:
No, I mean that also it might be system OpenCL used. As far as I remember, you can install OpenCL on Ubuntu by this command: `sudo apt install ocl-icd-opencl-dev`. I'm not sure, can we name this package as OpenCL SDK or not. But anyway it should be enough to build the `cpp_rpc` for the host with OpenCL support. OpenCL will be found on this [line](https://github.com/apache/tvm/pull/13362/files#diff-45747f6e0c43e6e7579d2c50160a4bdd25108cf04e98dc7ed8f1b36ddcad09b1R19) and we won't use the wrapper at all.

Reply:
Let's stick to OpenCL wrapper only. It will definitely find OpenCL installed by apt